app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/218760840-a8c076f2-e594-41c7-a3e4-0728631ca5fe.png)
fix this